lib/EasyDirectiveVisitor.ts

```diff
@@ -302,10 +308,110 @@ abstract class EasyDirectiveVisitor<
   ): DocumentNode[] {
     const unknownTypes: GraphQLNamedType[] = [];
     this.commonTypes.forEach(type =>
+      // @ts-expect-error (FIXME: type is not assignable)
       collectUnknownNamedTypes(schema, type, unknownTypes),
     );
     return unknownTypes.map(type => gql(printType(type)));
   }
+
+  public abstract visitArgumentDefinition(
+    argument: GraphQLArgument,
+    { field }: { field: GraphQLField<unknown, unknown> },
+    directiveArgs: TArgs,
+  ): void;
+
+  public abstract visitInputObject(
+    object: GraphQLInputObjectType,
+    directiveArgs: TArgs,
+  ): void;
+
+  public abstract visitInputFieldDefinition(
+    field: GraphQLInputField,
+    {
+      objectType,
+    }: {
+      objectType: GraphQLInputObjectType;
+    },
+    directiveArgs: TArgs,
+  ): void;
+
+  public abstract visitFieldDefinition(
+    field: GraphQLFieldConfig<unknown, TContext, TArgs>,
+    {
+      objectType,
+    }: {
+      objectType: GraphQLObjectType;
+    },
+    directiveArgs: TArgs,
+  ): void;
+
+  public abstract visitObject(
+    object: GraphQLObjectType | GraphQLInterfaceType,
+    directiveArgs: TArgs,
+  ): void;
+
+  abstract visitQuery(
+    query: GraphQLObjectType<any, TContext>,
+    schema: GraphQLSchema,
+    directiveName: string,
+  ): void;
+}
+
+// TODO: Felipe: is this class really necessary?
+/* eslint-disable class-methods-use-this, class-methods-use-this, @typescript-eslint/no-unused-vars */
+// istanbul ignore next
```

Comment:
i don't think we need this, just add these to the class `EasyDirectiveVisitor`

Reply:
removed